live-examples/js-examples/array-includes.html

```diff
@@ -1,12 +1,15 @@
 <pre>
-<code id="static-js">// create an array
-var arr1 = [1, 2, 3];
+<code id="static-js">var array1 = [1, 2, 3];
 
-// call includes() on the array passing a searchElement
-var result = arr1.includes(2);
+console.log(array1.includes(2));
+// expected output: true
+
+var array2 = ['cat', 'dog', 'bat'];
 
-// log the result
-console.log(result);
+console.log(array2.includes('cat'));
 // expected output: true
+
+console.log(array2.includes('at'));
+// expected output: false
```

Comment:
in a case like this, the numeric naming you've chosen here seems better (at least, it's hard to think of a descriptive name for the first array, and given a numeric name for the first, it seems better to have a numeric name for the second).